FABulous/FABulous_CLI/FABulous_CLI.py

```diff
@@ -1091,11 +1164,33 @@ def do_generate_custom_tile_config(self, args: argparse.Namespace) -> None:
     @with_category(CMD_FABRIC_FLOW)
     @with_argparser(tile_list_parser)
     def do_gen_io_tiles(self, args: argparse.Namespace) -> None:
+        """Generate I/O BELs for specified tiles.
+
+        This command generates Input/Output Basic Elements (BELs) for the
```

Comment:
Did you leave this in on purpose? I always though `BEL` means `Basic Element of Logic`?
```suggestion
        This command generates Input/Output Basic Elements of Logic (BELs) for the
```